### PR TITLE
Rename remaining identifiers to OpenSearch

### PR DESCRIPTION
**Pull request by VachaShah**
Signed-off-by: Vacha Shah <vachshah@amazon.com>

### Description
This PR updates the remaining identifiers from OpenDistro to OpenSearch.
 
### Issues Resolved
Part 3 of #19 
 
### Check List
- [ ] New functionality includes testing.
  - [X] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [X] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ logger.commons.level = debug
 
 ## Code of Conduct
 
-This project has adopted an [Open Source Code of Conduct](https://opendistro.github.io/for-elasticsearch/codeofconduct.html).
+This project has adopted an [Open Source Code of Conduct](https://github.com/opensearch-project/OpenSearch/blob/main/CODE_OF_CONDUCT.md).
```
Comment (tlfeng) [inline README.md:41]: Here is also a page of "Code of Conduct": https://www.opensearch.org/codeofconduct.html, but either is fine. 😁

Comment (VachaShah) [reply README.md]: I actually updated this section to how it is mentioned in [CONTRIBUTING.md](https://github.com/opensearch-project/common-utils/blob/main/CONTRIBUTING.md#code-of-conduct) of this repository to keep it consistent.

Comment (tlfeng) [reply README.md]: Ah, sounds like a new idea 👍

Comment (dblock) [reply README.md]: While ours may be identical to that (I think), we tend to keep a copy to avoid external links changing on us.

**Review by dblock — APPROVED** [review]

**Comment by dblock** [comment]
For standardizing CoC section I opened https://github.com/opensearch-project/.github/issues/8

**Pull request merged**